tools/db/move-team/dump_merge_teams.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# modify dump so that all teams are merged into one
+
+set -eu
+
+dir="$1"
+
+script=$(
+    for row in $(cat "$1/galley.team" | jq '.[0]' | uniq); do
+        echo "s/$row/\"e09f7a63-b5d4-4db4-a3c1-18bddf3df7fc\"/g;"
+    done
+)
+
+for f in $dir/*; do
+    echo $f
+    sed -e "$script" -i $f
+done
```

Comment:
hehe, nice.  took me a moment to guess what you did with that, but yes, i want this1  :)